apps/atr-one/app/controllers/races_controller.rb

```diff
@@ -69,6 +69,21 @@ def pubnub_config
                                                   .slice(:subscribe_key, :user_id)
                                                   .merge(channel: 'at-the-races')
                                                   .symbolize_keys
-                                                  # .transform_keys { |key| "pubnub_#{key}".to_sym }
+  end
+
+  def centrifugo_config
+    @centrifugo_config ||= build_centrifugo_config
+  end
+
+  def build_centrifugo_config
```

Comment:
Note that the centrifugo URL for client (browser) connection has to be specified. In production, this would be running on an accessible server somewhere.

Reply:
If you move this installation to a different server, this URL will have to be changed.